src/system_wrappers/source/cpu_info.cc

```diff
@@ -45,6 +48,10 @@ static int DetectNumberOfCores() {
   }
 #elif defined(WEBRTC_FUCHSIA)
   number_of_cores = zx_system_get_num_cpus();
+#elif defined(WEBRTC_FREEBSD)
```

Comment:
google says freebsd has _SC_NPROCESSORS_ONLN used by the WEBRTC_LINUX codepath, isn't it better to use it on FeeBSD, as well?